src/content/blog/2024-04-10-the-telemetry-tale.md

```diff
@@ -0,0 +1,116 @@
+
+E.g.
+| Raw Value | Bucket Size | Rounded Value |
+|-----------|-------------|---------------|
+| 8300 | 10000 | 10000 |
+| 13200 | 15000 | 15000 |
+| 147700 | 160000 | 160000 |
+| 1426100 | 2560000 | 2560000 |
+| 1788200 | 2560000 | 2560000 |
+| 90422400 | 10000000 | 90000000 |
+| 112400400 | 10000000 | 100000000 |
+| 222290500 | 10000000 | 100000000 |
+
+#### Local Differential Privacy
+
+Fully activating nerd mode, we read up on Local Differential Privacy (LDP). LDP is a variant of differential privacy where noise is added to each individual's data point locally before it is made available for collection. Once the data has been rounded, we apply a random amount of noise to each data point. This noise is derived from the Laplacian distribution which is generated based on a privacy parameter that is relative to the size of the rounded value. This distribution is characterized by its sharp peak and exponential decay, which means that most of the noise added to the data will be relatively small, thus minimally impacting the overall utility of the data, while providing a strong guarantee of privacy.
+
+Central to our privacy framework is the empowerment of ASEs with the informed ability to opt-in or opt-out of the telemetry service. We also only collect data on outgoing payments, thus honoring the preferences of those who may choose to disable telemetry. In other words if one ASE enables telemetry and receives payments from an entity who has not enabled telemetry we won’t collect metrics on those incoming payments. 
+
+For more information please have a look at our privacy [docs](https://github.com/interledger/rafiki/blob/main/packages/documentation/src/content/docs/telemetry/privacy.md) and our [implementation](https://github.com/interledger/rafiki/blob/main/packages/backend/src/telemetry/privacy.ts).
+
+### Architecture and Instrumentation
+
+![The Rafiki telemetry architecture](/developers/img/blog/2024-04-10/telemetry-architecture.png)
+
+The [number of transactions](https://github.com/interledger/rafiki/blob/d3be6b8d151d8cebc32b862e52a7bb678674d48e/packages/backend/src/open_payments/payment/outgoing/lifecycle.ts#L84-L90) is extracted from the Open Payments outgoing payment lifecycle and the value metric is handled by a [telemetry middleware](https://github.com/interledger/rafiki/blob/d3be6b8d151d8cebc32b862e52a7bb678674d48e/packages/backend/src/payment-method/ilp/connector/core/middleware/telemetry.ts) layer inside the ILP connector core.
+
+In summary, before these metrics are sent to the Otel collectors we’ve implemented local privacy-preserving measures, including:
```

Comment:
Comma between "...to the Otel collectors, we've implemented..."

Reply:
```suggestion
In summary, before these metrics are sent to the Otel collectors, we’ve implemented local privacy-preserving measures, including:
```